### DISCUSSION TITLE
What is the last recommended version of the plugin to work with Alfred v.4 ?

### DISCUSSION THREAD

**Discussion by itoldusoandso** [chrisgrieser/shimmering-obsidian#178] [General]
Obsidian is 1.5.12
Alfred is 4.6.6
What is the last version of the plugin compatible with Alfred?
I tried but I can't get anything working.

https://forum.obsidian.md/t/alfred-with-obsidian-shimmering-obsidian-plugin-need-help-getting-this-working/83273

Don't know what I am doing wrong.

**Comment by chrisgrieser** [comment]
The current workflow works with newer versions of Alfred *and* Obsidian. Past releases work with *older* versions of Alfred *and* Obsidian. I think that combination of versions makes it very tricky – you are using a *newer* version of Obsidian with an *older* version of Alfred.

To be honest, I also do not know whether this works – it might be a hard trial-and-error, if there is any version at all that works, sorry.

**Comment by itoldusoandso** [comment]
Well, it's okay, I am not asking to solve my problem, and since I use a free software, I wouldn't complaining at all ;)
But I wonder if I can get my hands on hold version of the plugins. I just wanna download, instal and test, to see if I can find one that works. If I fail, I accept and move on. Is there a repository where I can find the old versions, like github find the releases? Thank you.

**Reply by chrisgrieser** [reply]
Oh I see. If you just wanna give it a try, all of versions can be found at the release page here: https://github.com/chrisgrieser/shimmering-obsidian/releases

**Comment by itoldusoandso** [comment]
Thank you.
One more question .. what do I put in the 3rd step:

"Set your vault path in the workflow configuration.”

I don’t see in the instructions whether to use this or something else:

vault_path
/Users/USERNAME/Obsidian/VAULTNAME

Is this right?
Thank you.




On Monday, June 10, 2024, Chris Grieser ***@***.***> wrote:
> Oh I see. If you just wanna give it a try, all of versions can be found
at the release page here:
https://github.com/chrisgrieser/shimmering-obsidian/releases
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.<
https://ci3.googleusercontent.com/meips/ADKq_NYgG6gO77B4FN7yi0zMN0m_-RVk5iGx-IlYfe1Ntaf5KeqTeDqOBeHklMfoVvP9hc0gOJ91gNX7k9UCkExcQq8tR3sq7198EkcE5HDX9nINVnMw30AqCLMldNSJ-ZDfvPyqEiYZBHGzbe57tHhI9Zp7UJc5jbZKCKKRuhND50VevhLoMZrPkDvTi09rirnJUHb655Z3yoJCc7dPABCVD4AflC52fDqqVF5fUSUS2ADh2hRzdXFwxTykUvahcW_l8Q=s0-d-e1-ft#https://github.com/notifications/beacon/ARFDDSKGLZD44FTMJDLWGMLZGUWMNA5CNFSM6AAAAABJAC4AL6WGG33NNVSW45C7OR4XAZNRIRUXGY3VONZWS33OINXW23LFNZ2KUY3PNVWWK3TUL5UWJTQASROJU.gif>Message
ID: <chrisgrieser/shimmering-obsidian/repo-discussions/178/comments/
***@***.***>

**Comment by chrisgrieser** [comment]
yes, `/Users/USERNAME/Obsidian/VAULTNAME` is correct. But depending on the version of the workflow, where and how exactly that variable is stored differs, so I cannot give you an exact universal answer
